charts/rancher-istio/106.0.0+up1.22.1/Chart.yaml

```diff
@@ -1,24 +1,24 @@
 annotations:
   catalog.cattle.io/certified: rancher
   catalog.cattle.io/display-name: Istio
-  catalog.cattle.io/kube-version: '>= 1.26.0-0 < 1.30.0-0'
+  catalog.cattle.io/kube-version: '>= 1.27.0-0 < 1.31.0-0'
```

Comment:
Yeah - I actually see my confusion now. @adamkpickering is right that this max version isn't high enough for 2.11.

As I mentioned I don't think there's an issue if the lower k8s version is even lower than the minimum 2.11 will support. However, there would be an issue if we don't support the maximum version of k8s that 2.11 will have.

Unless this is an intentional choice to limit k8s support scope for this chart based on upstream istio requirements. If that's the case then this point is moot - otherwise, this should be `< 1.33.0-0` instead of 1.31.

Reply:
It was intentional, Istio 1.22 does not [support](https://istio.io/latest/docs/releases/supported-releases/#support-status-of-istio-releases) 1.31 or 1.32.
My understanding is that the chart can be included as long as it supports any k8s version supported by that particular version of Rancher. It won't show up in the UI unless the user is using a version specified in the annotation. I.e. it will show for a user using Rancher v2.11 + k8s 1.30 but won't for a user using Rancher v2.11 + k8s 1.31